pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "vuer"
-version = "0.0.26-rc1"
+version = "0.0.26-rc2"
```

Comment:
right now I usually keep the version the same. I just update the github tag with a new hash.